src/main/scala/io/iohk/ethereum/domain/Blockchain.scala

```diff
@@ -290,7 +309,15 @@ class BlockchainImpl(
     evmCodeStorage.put(hash, evmCode)
 
   override def saveBestKnownBlock(number: BigInt): Unit = {
```

Comment:
In the BlockImport there are various places where this method is called, should we do the same change we did on save here and also change that call to (maybe better with an auxiliar function here):
```
    if (saveAsBestBlock && block.hasCheckpoint) {
      saveBestKnownBlockAndLatestCheckpointNumber(block.header.number, block.header.number)
    } else if (saveAsBestBlock) {
      saveBestKnownBlock(block.header.number)
    }
```